docs/implementation.md

```diff
@@ -61,9 +63,157 @@ records such that
 
 In this storage, index updation is a no-op, since the bloom filters are already written as a part of committing data.
 
+In the case of Copy-On-Write, a single parquet file constitutes one `file slice` which contains one complete version of 
+the file 
+
+{% include image.html file="hoodie_log_format_v2.png" alt="hoodie_log_format_v2.png" max-width="1000" %}
+
 #### Merge On Read
 
-Work in Progress .. .. .. .. ..
+In the Merge-On-Read storage model, the processes(spark application) for ingesting data (both inserts/updates) into the dataset
```

Comment:
May be we don't want to say "processes" or the fact that they are decoupled because people may choose to run inline compactions or something similar, can we say that in this model we choose to defer merging/re-writing updates ?